src/Donations/Endpoints/ListDonations.php

```diff
@@ -189,7 +191,26 @@ public function getDonations(): array
      */
     public function getTotalDonationsCount(): int
     {
-        $query = give()->donations->prepareQuery();
+        $builder = new ModelQueryBuilder(Donation::class);
```

Comment:
Same comment as before about not using the `ModelQueryBuilder`

Reply:
Resolved: e13dd57ce3cc1184463bf52711077f0c3b212542